localMatchApp/signup.cpp

```diff
@@ -0,0 +1,39 @@
+#include "signup.h"
+#include "ui_signup.h"
+#include "QDesktopWidget"
+#include <QMessageBox>
+
+SignUp::SignUp(QWidget *parent) :
```

Comment:
according with the coding standard, function identifiers must have identifier names that begin with lowercase letters. Also, arguments whose input value is used by the function, and whose value is not changed by the function should be prefixed by "i_" (i.e. input).  

Please apply this comment in all your files.